From 4ed2456e9883dcdf946abd306b558c5d60d2e8cf Mon Sep 17 00:00:00 2001
From: Aleksandr Korolev <aleksandr.korolev@auriga.com>
Date: Fri, 10 Jun 2022 17:46:59 +0300
Subject: [PATCH] Add some includes

---
 configure                 | 2 +-
 libavcodec/v4l2_request.h | 3 +++
 2 files changed, 4 insertions(+), 1 deletion(-)

diff --git a/configure b/configure
index 1c1929d..4e3e868 100755
--- a/configure
+++ b/configure
@@ -3041,7 +3041,7 @@ vp9_dxva2_hwaccel_deps="dxva2 DXVA_PicParams_VP9"
 vp9_dxva2_hwaccel_select="vp9_decoder"
 vp9_nvdec_hwaccel_deps="nvdec"
 vp9_nvdec_hwaccel_select="vp9_decoder"
-vp9_v4l2request_hwaccel_deps="v4l2_request"
+vp9_v4l2request_hwaccel_deps="v4l2_request vp9_v4l2_request"
 vp9_v4l2request_hwaccel_select="vp9_decoder"
 vp9_vaapi_hwaccel_deps="vaapi VADecPictureParameterBufferVP9_bit_depth"
 vp9_vaapi_hwaccel_select="vp9_decoder"
diff --git a/libavcodec/v4l2_request.h b/libavcodec/v4l2_request.h
index 58d2aa7..fec7956 100644
--- a/libavcodec/v4l2_request.h
+++ b/libavcodec/v4l2_request.h
@@ -20,6 +20,9 @@
 #define AVCODEC_V4L2_REQUEST_H
 
 #include <linux/videodev2.h>
+#include <linux/v4l2-controls.h>
+#include <sys/ioctl.h>
+#include <unistd.h>
 
 #include "libavutil/hwcontext_drm.h"
 
-- 
2.36.1

